setup.cfg

```diff
@@ -0,0 +1,27 @@
+[bdist_wheel]
+universal=1
+
+[metadata]
+name = nmigen
+author = whitequark
+author_email = whitequark@whitequark.org
+description = Python toolbox for building complex digital hardware
+license = BSD
+project_urls =
+    Source Code=https://github.com/nmigen/nmigen
+    Bug Tracker=https://github.com/nmigen/nmigen/issues
+
+[options]
+python_requires = ~=3.6
+install_requires =
+    pyvcd~=0.2.0
+    Jinja2~=2.11
+packages = find:
+
+[options.entry_points]
+console_scripts =
+    nmigen-rpc = nmigen.rpc:main
+
+[tools.setuptools_scm]
+version_scheme = guess-next-dev
+local_scheme = local_scheme.local_scheme
```

Comment:
I don't think this is correct, might need to be `attr: local_scheme.local_scheme`

Reply:
Can we inline the Python code and get rid of yet another auxiliary file? It's not very important, but going from one file to four with the exact same info is annoying.

Reply:
No - that's the point of setuptools declarative though. To avoid python code as much as possible

Reply:
OK, makes sense.